stylix/palette.nix

```
@@ -117,19 +117,16 @@ in
       '';
       type =
         with lib.types;
-        nullOr (oneOf [
+        oneOf [
           path
           lines
           attrs
-        ]);
-      # defaults to null when cfg.image is also null, in order to trigger the assertion below correctly
-      default = if cfg.image == null then null else cfg.generated.palette;
+        ];
       defaultText = lib.literalMD ''
-        The colors used in the theming.
-
-        Those are automatically selected from the background image by default,
-        but could be overridden manually.
+        Automatically selected from the background image. However, no default
+        is provided if `stylix.image` is null.
```

If so, this text would also need to change:

```suggestion
        Automatically selected from the background image. The default value will
        throw if `stylix.image` is null.
```